pkg/applicationserver/applicationserver.go

```diff
@@ -553,6 +553,8 @@ func (as *ApplicationServer) handleUp(ctx context.Context, up *ttnpb.Application
 		return true, as.decryptDownlinkMessage(ctx, up.EndDeviceIdentifiers, p.DownlinkAck, link)
 	case *ttnpb.ApplicationUp_DownlinkNack:
 		return true, as.handleDownlinkNack(ctx, up.EndDeviceIdentifiers, p.DownlinkNack, link)
+	case *ttnpb.ApplicationUp_ServiceData:
+		return true, nil
 	default:
```

Comment:
Or we make the default `true` ?

Reply:
No I think it's good to be explicit here what we send upstream. We have a compatibility commitment towards applications, which is basically controlled here.